src/lib/src/component/navigation/navigation.component.spec.ts

```diff
@@ -384,4 +385,69 @@ describe('NavigationComponent', () => {
     expect(runIcon.properties['disabled']).toBeFalsy;
   }));
 
+  it('displays notification when test execution has been started', fakeAsync(() => {
+    // given
+    setupWorkspace(component, fixture);
+    component.uiState.selectedElement = tclFile;
+    fixture.detectChanges();
+    let runIcon = sidenav.query(By.css('#run'));
+
+    // when
+    runIcon.nativeElement.click();
+    tick();
+
+    // then
+    fixture.detectChanges();
+    let notify = fixture.debugElement.query(By.css('#notification'));
+    expect(notify).toBeTruthy();
+    expect(component.notification).toEqual(`Execution of ${tclFile.name} has been started.`);
+    expect(notify.nativeElement.innerText).toEqual(component.notification);
+
+    tick(4000);
```

Comment:
this tick is probably only there to cleanup the message, right? wouldn't it be better to not wait (and slow down the test) and actively reset?
How about setting the timeout during the test to less than 4 seconds (in this case you can't use a constant of course, see my previous comment)?

Reply:
Right, there's actually a `flush()` method provided by Angular that removes all remaining microtasks from the queue in a fakeAsync block. That should make the test more readable. Its own documentation is pretty clear on what it does, so I refrained from adding an additional comment.